src/FileService.Validation/ImageRequestValidator.cs

```diff
@@ -0,0 +1,25 @@
+﻿using FluentValidation;
+using LT.DigitalOffice.FileService.Models.Dto.Requests;
+using LT.DigitalOffice.FileService.Validation.Helpers;
+using System;
+using System.Collections.Generic;
+
+namespace LT.DigitalOffice.FileService.Validation
+{
+    public class ImageRequestValidator : AbstractValidator<ImageRequest>
+    {
+        public static List<string> AllowedExtensions { get; } =
+            new List<string> { ".jpg", ".jpeg", ".png", ".gif", ".bmp", ".tga" };
```

Comment:
What's the catch to use a static property here and with one more get?

Reply:
Fixed.